pkg/client/unversioned/testclient/simple/simple_testclient.go

```diff
@@ -86,6 +88,8 @@ func (c *Client) Setup(t *testing.T) *Client {
 			Host:         c.server.URL,
 			GroupVersion: testapi.Extensions.GroupVersion(),
 		})
+
+		c.Clientset = release_1_1.AdaptOldClient(c.Client)
```

Comment:
Why are we doing this?
Do we have a latest.Clientset or head.Clientset?
release_1_1.Clientset seems misleading since this is not really a release 1.1 client set.